apps/condo/domains/billing/schema/BillingAccount.test.js

```diff
@@ -1,343 +1,391 @@
-
-        expect(billingAccount.context.id).toEqual(context.id)
-        expect(billingAccount.property.id).toEqual(property.id)
-    })
-
-    test('user: create BillingAccount', async () => {
-        const admin = await makeLoggedInAdminClient()
-        const { context } = await makeContextWithOrganizationAndIntegrationAsAdmin()
-        const [property] = await createTestBillingProperty(admin, context)
-        const client = await makeClientWithNewRegisteredAndLoggedInUser()
+    describe('Constraints', () => {
+        test('can create two BillingAccount with same globalId in different contexts', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const { context } = await makeContextWithOrganizationAndIntegrationAsAdmin()
+            const [property] = await createTestBillingProperty(admin, context)
+            const [billingAccount] = await createTestBillingAccount(admin, context, property, {
+                globalId: 'cat',
+            })
+
+            const { context: context2 } = await makeContextWithOrganizationAndIntegrationAsAdmin()
+            const [property2] = await createTestBillingProperty(admin, context2)
+            const [billingAccount2] = await createTestBillingAccount(admin, context2, property2, {
+                globalId: 'cat',
+            })
+
+            expect(billingAccount.id).not.toEqual(billingAccount2.id)
+            expect(billingAccount.globalId).toEqual(billingAccount2.globalId)
+        })
 
-        await expectToThrowAccessDeniedErrorToObj(async () => {
-            await createTestBillingAccount(client, context, property)
+        test('cant create two BillingAccount with same globalId in single context', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const { context } = await makeContextWithOrganizationAndIntegrationAsAdmin()
+            const [property] = await createTestBillingProperty(admin, context)
+            await createTestBillingAccount(admin, context, property, {
+                globalId: 'cat',
+            })
+            
+            const [property2] = await createTestBillingProperty(admin, context)
+            
+            await catchErrorFrom(
+                async () =>
+                    await createTestBillingAccount(admin, context, property2, {
+                        globalId: 'cat',
+                    }),
+                (e) => {
+                    expect(e.errors).toBeDefined()
+                }
+            )
         })
     })
 
-    test('anonymous: create BillingAccount', async () => {
-        const admin = await makeLoggedInAdminClient()
-        const { context } = await makeContextWithOrganizationAndIntegrationAsAdmin()
-        const [property] = await createTestBillingProperty(admin, context)
-        const client = await makeClient()
+    describe('Create', () => {
+
+        test('admin: create BillingAccount', async () => {
```

Comment:
This naming is more natural to read
```suggestion
        it('can be created by admin', async () => {
```